pg_analytics/README.md

```diff
@@ -76,14 +76,14 @@ As `pg_analytics` becomes production-ready, many of these will be resolved.
 
 ### Install Rust
 
-To develop the extension, first install Rust v1.76.0 using `rustup`. We will soon make the extension compatible with newer versions of Rust:
+To develop the extension, first install Rust v1.77.2 using `rustup`. We will soon make the extension compatible with newer versions of Rust:
```

Comment:
You can remove the "newer versions of Rust". v1.77.2 is the latest version, this was an artifact of being stuck on an old Rust version earlier due to Tantivy